c-api-examples/decode-file-c-api.c

```diff
@@ -5,23 +5,70 @@
 // This file shows how to use sherpa-onnx C API
 // to decode a file.
 
-#include <getopt.h>
+#include <cargs.h>
```

Comment:
```suggestion
#include "cargs.h"
```

Please use `""` for non-system headers.

(You may need to sort the headers alphabetically after changing `<` to `"`)